test/src/requestinterception.spec.ts

```diff
@@ -867,7 +867,14 @@ describe('request interception', function () {
       });
       expect(response.status()).toBe(200);
       expect(response.headers()['foo']).toBe('bar');
-      expect(response.headers()['arr']).toBe('1\n2');
+
+      // The separator used to handle headers with multiple values is browser
```

Comment:
Do you have a link to a spec that defines how it is supposed to be handled?

Reply:
I thought I replied here yesterday, sorry.

So the main source I have is https://www.rfc-editor.org/rfc/rfc9110.html#section-5.2
Which explains that combined header values should be CSV. However it allows for exceptions such as for the Set-Cookie header (see the note in https://www.rfc-editor.org/rfc/rfc9110.html#section-5.3)

I think Firefox happens to use \n separators for Set-Cookie since `,` can't be used. However I don't know why Chrome would use \n all the time, it doesn't seem spec compliant.

Reply:
Interesting, in DevTools I see that Chrome also uses a comma

Reply:
Would you mind leaving a note in the test that Chrome is likely not compliant. I have briefly investigated but I could not figure out where it goes wrong.

Reply:
Updated!